test/SimpleWebRequest.spec.ts

```diff
@@ -117,10 +120,14 @@ describe('SimpleWebRequest', () => {
         const method = 'GET';
         const url = faker.internet.url();
         const error = `Don't set Accept with options.headers -- use it with the options.acceptType property`;
-        const request = new SimpleWebRequest<string>(url, method, {}, () => headers);
+        const request = new SimpleWebRequest<string>(method, url, {}, () => headers);
 
-        expect(() => request.start()).toThrowError(error);
-        expect(console.error).toHaveBeenCalledWith(error);
+        request.start().then(() => {
+            expect(false).toBeTrue();
```

Comment:
Nit: This file has `fail()` so either use that instead of each of these `expect(false).toBeTrue()` or replace the usage of `fail()` (if it doesn't work with jasmine or something).